README.md

```diff
@@ -19,102 +19,34 @@
   </a>
 </p>
 
-MUD is a framework for complex Ethereum applications.
-
-It adds some conventions for organizing data and logic and abstracts away low-level complexities so you can focus on the features of your app.
-
-It standardizes the way data is stored on-chain.
-With this standard data model, MUD can provide all network code to synchronize contract and client state. This includes synchronizing state directly from an RPC node or a general-purpose MUD indexer.
+MUD is a framework for ambitious Ethereum applications. It compresses the complexity of building EVM apps with a tightly integrated software stack.
 
 MUD is MIT-licensed, open source and free to use.
 
 ## Features
 
+<!--
 ![MUD features](./docs/public/features.png)
+-->
 
-### Today
-
-- State synchronization between contracts and clients without custom networking code
-- General purpose indexers (without custom indexing code)
-- Seamless contract upgrades (+ automatic contract upgrades during development)
-- Shared contract state
-- Optimistic updates
-- Automatic type generation for contracts and systems
-- Query language to interact with contract state
-- Data explorer to inspect and modify contract and local state
-- Bitpacking utilities
-
-### Soon
-
-- Local simulation of transactions (including optimistic state)
-- Built-in support for account abstraction
-- Contract package manager
+See [the development status page](https://status.mud.dev/).
 
 ## Quickstart
 
 ```
 pnpm create mud@next my-project
 ```
 
-For more information on how to get started, have a look at the [MUD documentation](https://mud.dev/quick-start).
-
-## Talks
-
-<div>
-  <a href="https://www.youtube.com/watch?v=UvIWmzscWp8" target="_blank"><img src="https://img.youtube.com/vi/UvIWmzscWp8/hqdefault.jpg" alt="Getting started with MUD" width="30%" /></a>
-  <a href="https://www.youtube.com/watch?v=j-_Zf8o5Wlo" target="_blank"><img src="https://img.youtube.com/vi/j-_Zf8o5Wlo/hqdefault.jpg" alt="MUD - An Engine for Autonomous Worlds" width="30%" /></a>
-  <a href="https://www.youtube.com/watch?v=mv3jA4USZtg" target="_blank"><img src="https://img.youtube.com/vi/mv3jA4USZtg/hqdefault.jpg" alt="MUD in Practice - How we built OPCraft" width="30%" /></a>
-</div>
```

Comment:
can we make a docs page for this? sad to not have a point of reference for videos on where to learn more about MUD

Reply:
We have a link to the youtube channel on the home page. Can you think of a better place? 

<img width="1419" alt="Screenshot 2024-02-07 at 3 39 56 PM" src="https://github.com/latticexyz/mud/assets/12722969/9dad8784-1968-441f-a852-95b8f542e08f">

Reply:
oh I mean it would be nice to have a gallery of handpicked videos that cover different topics, instead of asking folks to find the youtube link and dig through the video list

Reply:
Either that, or we can integrate the videos into our regular docs based on their topics. I think that would be better than a separate gallery - people don't want a video or a text, they want information about a specific topic, IMAO.

Can he hash this out at https://discord.com/channels/865335009915961364/1051926481917456384/1204913514104225892 ?